sdk/go/common/apitype/policy.go

```diff
@@ -46,6 +46,9 @@ type CreatePolicyPackRequest struct {
 type CreatePolicyPackResponse struct {
 	Version   int    `json:"version"`
 	UploadURI string `json:"uploadURI"`
+	// RequiredHeaders represents headers that the CLI must set in order
+	// for the upload to succeed.
+	RequiredHeaders map[string]string `json:"headers,omitempty"`
```

Comment:
We typical try to keep the json and go property names aligned modulo Pascal vs camel casing. Should this just be `Headers`?

Reply:
oh shoot. I totally overlooked naming the json property name after my IDE auto-filled the `omitempty`. But yes, I'll camel-case this.